dashboard/apps/prototype/models.py

```diff
@@ -178,7 +178,7 @@ def profile(self, start_date=None, end_date=None, freq='MS'):
             'beta_date': self.beta_date,
             'live_date': self.live_date,
             'end_date': self.end_date,
-            'spendings': {}
+            'financial': {}
```

Comment:
if we can trust this page http://wikidiff.com/financials/financial, it should be `financials` instead of `financial`.